app/views/advanced_messenger/_history_journal_bottom.html.erb

```diff
@@ -0,0 +1,151 @@
+                        + (isReadByCurrentUser ? 
+                        "<a class='icon icon-checked-green readButton' data-remote='true' rel='nofollow' data-method='post' href='/advanced_messenger/" + id + "/0/update_read_by_users'></a>" 
+                      : "<a class='icon icon-unchecked-red unreadButton' data-remote='true' rel='nofollow' data-method='post' href='/advanced_messenger/" + id + "/1/update_read_by_users'></a>"));
+        }
+    }
+
+    const id = "<%= journal.id %>";
+    const read_by_users = '<%= journal.read_by_users.html_safe %>';
+    const notesPreview = "<%= escape_javascript(journal.notes)[0, 50] %>"
+
+    var currentNote = $("#change-" + id + ".journal.has-notes");
+    if (currentNote.length != 0) {
+        // ====== Set the note panel blue border ======
+        currentNote.addClass("advanced-messenger-note");
+
+        // ====== Set the placeholders that will contain dynamically render content (that changes without page refresh) ======
+        currentNote.find('.contextual').after("<div class='read-unread-indicators-at-right' style='float: right;'><span class='for_current_user_placeholder'></span></div>");
+        const gravatar = currentNote.find(".gravatar");
+        const expandCollapse = "<div class='expand-collapse-icon-placeholder'></div>";
+        if (gravatar.length == 0) {
+            currentNote.find(".note-header").prepend(expandCollapse);
+        } else {
+            gravatar.after(expandCollapse);
+        }
+
+        currentNote.find('.wiki').append("<p class='note-preview'>" + notesPreview + "... </p>");
+
+        // ====== Expand/collapse indicators and panel + read indicator for currrent user ======
+        renderExpandEndReadIndicatorForCurrentUser(id, read_by_users, currentUserId, notesPreview, true);
+
+        // ====== Read/Unread indicators and tooltip for other users than the logged in user ======
+        <% read_by_users = JSON.parse(journal.read_by_users) %>
+        let userCapitals;
+        let color;
+        let tooltipUserList = "";
+        let readByOthers = "<span style='padding-right: 5px'>";
+        let wereOtherNotified = false;
+        <% read_by_users.keys.each_with_index do |user_id, index| %>
+            <% if user_id != User.current.id.to_s %>
+                wereOtherNotified = true;
+                <%user = User.find(user_id)%>
+                <%read_by_user = read_by_users[read_by_users.keys[index]]%>
+                userCapitals = "<%=user.firstname%>".charAt(0).toUpperCase() + "<%=user.lastname%>".charAt(0).toUpperCase(); 
+                colorStyle = "style='color:" + (<%=read_by_user["read"] == 0%> ? RED : GREEN) + "'";
+                tooltipUserList += "<li " + colorStyle.replace(GREEN, "green") + ">" + userCapitals + "-<%=user.firstname%> <%=user.lastname%>" + " (<%=read_by_user["read"] == 0 ? t(:journal_notifications_tooltip_list_item_not_viewed) : read_by_user["date"].to_date.strftime("%d %b %Y %H:%M")%>)</li>";
```

Comment:
Probably the date format should be taken from user settings. I think there is an item in 32483 as well.

Reply:
I I saw that in other places in redmine, the ```format_time()``` function is used to display dates with time, that internally uses:
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/abb299fb-9dc2-47ec-b05b-ac6f1b2a89db)
I changed my code to also use this function: 
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/123e0b18-090b-4f57-85cd-d37f32546f9c)